app/controllers/api/encryptables_controller.rb

```diff
@@ -18,34 +17,37 @@ def index
   # GET /api/encryptables/:id
   def show
     authorize entry
-    entry.decrypt(decrypted_team_password(team))
+
+    if entry.transferred?
+      personal_team = current_user.personal_team
+      personal_team_password = decrypted_team_password(personal_team)
+      EncryptableTransfer.new.receive(entry, session[:private_key], personal_team_password)
```

Comment:
In our request to show a transferred encryptable, we don't send `Authorization-Password` in the headers. The method users_private_key uses these headers, and this is here nil. Because we use `session[:private_key]` multiple places, it should be okey like that, right?